WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -914,8 +925,12 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
 
     fileprivate func configureLikeActionButton(_ animated: Bool = false) {
         likeButton.isEnabled = ReaderHelpers.isLoggedIn()
+        // as by design spec, only display like counts
+        let likeCount = post?.likeCount()?.intValue ?? 0
+        let shortTitle = likeCount > 0 ? "\(likeCount)" : ""
+
+        let title = FeatureFlag.postReblogging.enabled ? shortTitle : post!.likeCountForDisplay()
```

Comment:
I know this is legacy, but we _really_ try not to force unwrap. Can you fix both of these?

`post!.likeCountForDisplay()`
`post!.isLiked`

Reply:
Totally agree, even more so for the button title `post!likeCountForDisplay( )` which can also be set to nil, so it shouldn't be force unwrapped